raiden_contracts/deploy/__main__.py

```diff
@@ -548,6 +552,12 @@ def token(
 @click.option(
     '--registry-address',
     default=None,
+    callback=error_removed_option,
+    help='removed',
```

Comment:
I think using `hidden=True` instead of providing a help text would be nicer for removed commands.

Reply:
I go for `hidden=True`.  Thanks.